solutions/usaco-896.mdx

```diff
@@ -125,4 +125,33 @@ class mountain implements Comparable<mountain> {
 
 </JavaSection>
 
+<PySection>
+```python
+import sys
+
+sys.stdin = open("mountains.in", "r")
+sys.stdout = open("mountains.out", "w")
```

Comment:
(optional) could you use `with` instead?

Reply:
okay ill change all of them

Reply:
changed it